### DISCUSSION TITLE
FS22 Fox Farms

### DISCUSSION THREAD

**Discussion by frankg79** [KillBait/FS19_AutoDrive_MapImages#8] [General]
Hi,
is it possible to integrate FS22 Fox Farms?

Thanks,
Frank

**Comment by KillBait** [comment]
Hi, no problem, I'll add is as soon as I can

**Comment by KillBait** [comment]
The images are now added.

**Comment by frankg79** [comment]
Thanks! Sorry, i have another question now. The editor shows me, that the image is downloaded and loaded, but it isn't showed in the editor, even if it try to load it manually.. i'm a little confused and think, i forgotten to do something.. but i doesn't know what.. :/

**Comment by KillBait** [comment]
Sorry about that, It is a Editor bug, a new version should be uploaded in a few minutes

**Reply by frankg79** [reply]
Thanks, great work!!
